README.rst

```diff
@@ -115,7 +115,8 @@ To run SCT tests locally run following::
     # create a virtualenv for SCT
     pyenv virtualenv 3.8.3 sct38
     pyenv activate sct38
-    pip install -r requirements-python.txt
+    pip install -r requirements.txt
```

Comment:
as you mentioned `requirements.txt` is now a lock file, does pip "understands" lockfile format?

Reply:
yes, it plain `requirement.txt` format (that includes hashes), pip support it for quite some time now (few years)